.coafile

```diff
@@ -47,3 +47,112 @@ ignore = vendor/**, resources/vendors/**, _site/**
 bears = YAMLLintBear
 files = **.yml
 ignore = vendor/**
+
+[projects_markdown]
+files = _projects/**.md
+bears = LineLengthBear
+max_line_length = 80
+ignore = _projects/corobo_enhancement.md,
+	_projects/bear_updates.md,
+	_projects/README.md,
+	_projects/improve_installation.md,
+	_projects/commit_based_perftest.md,
+	_projects/automate_dependency_repology.md,
+	_projects/automate_vulture_whitelist.md,
+	_projects/integrate-pyflakes-AST.md,
+	_projects/rstcheck_with_better_sphinx_support.md,
+	_projects/nextgen_action_system.md,
+	_projects/implement_aspects.md,
+	_projects/debug_profile.md,
+	_projects/improve_diff_handling.md,
+	_projects/jetbrains-ide-plugin.md,
+	_projects/cobot_enhancement.md,
+	_projects/example.md,
+	_projects/aspects.md,
+	_projects/communication_bridge.md,
+	_projects/assign_issues.md,
+	_projects/coala_web.md,
+	_projects/use_coala.md,
+	_projects/enhance_coala_quickstart.md,
+	_projects/coala_cli.md,
+	_projects/go_bears_on_bsd.md,
+	_projects/gitmate_gitlab.md,
+	_projects/optimize_caching.md,
+	_projects/documentation_extraction.md,
+	_projects/git_task_list.md,
+	_projects/gitmate_jira.md,
+	_projects/use_coala_2.md,
+	_projects/gitmate_plugins.md,
+	_projects/use_coala_3.md,
+	_projects/vulture.md,
+	_projects/create_jenkins_plugin.md,
+	_projects/implement_metrics_for_coala.md,
+	_projects/newcomer_metrics.md,
+	_projects/editor_support.md,
+	_projects/extend_linter_integration.md,
+	_projects/pr_perftest.md,
+	_projects/add_repo_license.md,
+	_projects/antlr_v4.md,
+	_projects/coala_language_server.md,
+	_projects/gitmate_testing.md,
+	_projects/quickstart_green_mode.md,
+	_projects/commit-content-checks.md,
+	_projects/nested_languages.md,
+	_projects/meta_review.md,
+	_projects/convert_bear_to_aspects.md,
+	_projects/roberta_external_source.md,
+	_projects/generic_bears.md
+
+[projects_readability]
```

Comment:
a good section name will signify the checks it is doing and also any special reason for its creation

Reply:
eg `LineLength_future_files`

Reply:
Yes I knew that but couldn't think of a better name. Also is it appropriate to give section a long name like this one.

Reply:
never shy away from long descriptive names 😉